### PR TITLE
example of services file

### PR DESCRIPTION
**Pull request by Saphyel**
Example of how I think should be the *.services.yml

### CODE REVIEW

**Comment by Saphyel** [comment]
@FGM something like this

**Comment by fgm** [comment]
Not convinced (yet ?) : as you can see, you have to defining a fairly complex hierarchy, which is entirely environment-dependent, and in your example it goes alongside the module-defined (hence constant) logger channel configuration.

At the very least, if we choose to go that route, I think the module-provided configuration will have to be something constant which relies on DIC parameters, or an override, but such hardcoded settings as these would have no place in the module. And then if we have parameters, which need to go into services.yml, what exactly is stable/default enough that it can remain in the rabbitmq.services.yml proper ?

**Comment by Saphyel** [comment]
@FGM what about this way d35ac941730ccea6cc366ed1360ddc7f8b9a2689 ?

And well I don't think is complex hierarchy... is just separate the config (how is configured the rabbitmq) from your code...

File: tests/rabbitmq_example/rabbitmq_example.services.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      example:
+        connection: default
+        exchange_options:
+          name: 'dasp'
+          type: direct
+          passive: false
+          durable: true
+          auto_delete: false
+          internal: false
+          nowait: false
```
Comment (pvhee) [inline tests/rabbitmq_example/rabbitmq_example.services.yml:21]: @Saphyel You ideally would want to have multiple queues bound into a single exchange, so this export format wouldn't work I think

Comment (Saphyel) [inline tests/rabbitmq_example/rabbitmq_example.services.yml:21]: @pvhee it's the same config that uses this symfony bundle: https://github.com/videlalvaro/rabbitmqbundle and the author of that bundle works in rabbitmq organisation...
Also I test that scenario and works in that way.

Comment (pvhee) [inline tests/rabbitmq_example/rabbitmq_example.services.yml:21]: @Saphyel how would you create multiple queues and one exchange in this format?

Comment (Saphyel) [inline tests/rabbitmq_example/rabbitmq_example.services.yml:21]: @pvhee in this way

```
routing_keys: 
  - 'argo.route1'
  - 'argo.route2'
  - 'argo.queueloca'
```

**Pull request closed without merging**